test/stats/test_main_results.py

```diff
@@ -668,7 +668,7 @@ def test_profitable_weeks_no_trades_market_up():
 
 
 def test_profitable_weeks_no_trades_market_flat():
-    # One week, all days are flat - outcome should be draw for profitable week, draw for outperform
+    # One week, all days are flat - outcome should be drawn for profitable week, draw for outperform
```

Comment:
```suggestion
    # One week, all days are flat - outcome should be draw for profitable week, draw for outperform
```

Not all spellchecks should automatically be accepted ;)